test/output-consistency/mzcompose.py

```diff
@@ -0,0 +1,38 @@
+# Copyright Materialize, Inc. and contributors. All rights reserved.
+#
+# Use of this software is governed by the Business Source License
+# included in the LICENSE file at the root of this repository.
+#
+# As of the Change Date specified in that file, in accordance with
+# the Business Source License, use of this software will be governed
+# by the Apache License, Version 2.0.
+
+from materialize.mzcompose import Composition, WorkflowArgumentParser
+from materialize.mzcompose.services import Cockroach, Materialized
+from materialize.output_consistency.output_consistency import (
+    parse_output_consistency_input_args,
+    run_output_consistency_tests,
+)
+
+SERVICES = [
+    Cockroach(setup_materialize=True),
+    # We use mz_panic() in some test scenarios, so environmentd must stay up.
+    Materialized(propagate_crashes=False, external_cockroach=True),
```

Comment:
I think you're not using mz_panic here, so best to propagate crashes, then we can also use the ci-logging scripting to autodetect them.

Reply:
✅